tests/test_locales.py

```diff
@@ -61,11 +58,8 @@ def test_locales(self):
         assert len(locales._locales) > 0
 
 
+@pytest.mark.usefixtures("lang_locale_fixture")
 class TestLocale:
```

Comment:
Can this potentially be combined into `TestEnglishLocale` to remove the need for having the conditional in the `lang_locale_fixture`?

Reply:
yes, indeed. nice one.